mlflow/recipes/steps/transform.py

```diff
@@ -79,6 +79,15 @@ def _validate_and_apply_step_config(self):
                 "Missing target_col config in recipe config.",
                 error_code=INVALID_PARAMETER_VALUE,
             )
+        if "using" in self.step_config:
+            if self.step_config["using"] not in ["custom"]:
```

Comment:
I don't fully understand this change. Do we plan to add other forms of transformer option?

Reply:
Not right now, but this gives us way with which we can add more in future + provides consistency with other steps. Also custom is default here and no breaking change.
@mengxr can you add what exactly you had in mind?